jobs/v4/howto/basic_client_event_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestCreateClientEvent(t *testing.T) {
+	testutil.Retry(t, 10, 1*time.Second, func(r *testutil.R) {
+		tc := testutil.SystemTest(t)
+		buf := &bytes.Buffer{}
+
+		requestId := fmt.Sprintf("requestId-%d", time.Now().UnixNano())
+		eventId := fmt.Sprintf("eventId-%d", time.Now().UnixNano())
```

Comment:
Do these need to be globally unique? If so, can we use the `uuid` package (example: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/jobs/v3/howto/howto_test.go#L40).

Reply:
Yes, they should be unique. What's the advantage of uuid over time.Now()?